src/session.ts

```diff
@@ -223,7 +314,8 @@ export async function approveSpender(spender: string, amount: BigNumber, config:
 }
 
 export function disconnectWallet(): void {
-  window.localStorage.removeItem("session");
+  window.localStorage.removeItem('session');
+  window.localStorage.removeItem('walletconnect');
```

Comment:
Where is this key set?

Reply:
That's done automatically by walletConnect library

Reply:
Shouldn't we then use a library function to disconnect? How are we supposed to know that that is the key they use?

Reply:
the library function that i used previously is an event that takes in a string called "disconnect" and a callback function to tell the app to do something when it's disconnected which will still result to me clearing the localstorage.

Reply:
The only item that's being set to the localStorage in the app is the session and walletconnect. so when a user disconnect, it's appropriate to clear the session and the wallet connect instance. That's why i was using the window.localStorage.clear() method